assets/src/design-system/components/contextMenu/index.js

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import styled, { css } from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+/**
+ * Internal dependencies
+ */
+import { BUTTON_TRANSITION_TIMING } from '../button/constants';
+import { MenuItem, MenuItemProps } from './menu-item';
+
+const Popover = styled.div(
+  ({ isOpen, theme }) => css`
+    position: absolute;
+    display: none;
+    margin: 5px 0 0;
+    background-color: ${theme.colors.bg.primary};
+    pointer-events: none;
+    border-radius: ${theme.borders.radius.small};
+    border: 1px solid ${theme.colors.border.disable};
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+
+    ${isOpen &&
+    css`
+      display: block;
+      z-index: 10;
+      opacity: 1;
+      pointer-events: auto;
+    `};
+  `
+);
+
+const MenuContainer = styled.ul(
+  ({ theme }) => css`
+    background-color: ${theme.colors.bg.primary};
+    border-radius: ${theme.borders.radius.small};
+    margin: 0;
+    min-width: 210px;
```

Comment:
I'm seeing a width of 200px here: https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=5733%3A75504 which is what i _think_ is the base 'lego' for the right click menus. 

![Screen Shot 2021-02-11 at 1 29 18 PM](https://user-images.githubusercontent.com/10720454/107695003-42a0c680-6c6d-11eb-88f6-6642a7c78f3b.png)